media-player.js

```diff
@@ -1006,8 +1087,58 @@ class MediaPlayer extends FocusVisiblePolyfillMixin(InternalLocalizeMixin(RtlMix
 		}
 	}
 
+	_getMetadata() {
+		if (this.metadata) {
```

Comment:
Nit: invert to return early
```suggestion
		if (!this.metadata) {
		    return;
```